lib/coverband/utils/tasks.rb

```diff
@@ -9,11 +9,23 @@
     Coverband::Reporters::ConsoleReport.report(Coverband.configuration.store)
   end
 
+  if defined?(RubyVM::AbstractSyntaxTree)
+    desc "Output all dead methods"
+    task :dead_methods do
+      Coverband::Utils::DeadMethods.output_all
```

Comment:
I might just add the requires for the two things here since this checks for the required support and a user is trying to run it

Reply:
I like that idea. Will do.

Reply:
This is done.